apps/firefox/firefox.py

```diff
@@ -50,6 +50,14 @@ class BrowserActions:
     def bookmarks():
         actions.key(f"{cmd_ctrl}-shift-o")
 
+    def focus_address():
+        if app.platform == "mac":
+            actions.key("cmd-l")
+        else:
+            # Only using "ctrl-l" might fail and clear the console if the user
+            # is focused in the devtools
+            actions.key("f6 ctrl-l")
+
```

Comment:
I'm not sure about this - this is bound in a context matching `app: firefox`, but the default behaviors are bound in contexts matching `tag: browser` (or `tag: browser\nos: mac`). Neither of these contexts is strictly more specific than the other, so which one matches is kinda dependent on talon implementation details, which I'd rather not rely on.